cosmos-gui/conf/cosmos-gui.json

```diff
@@ -2,15 +2,27 @@
   "gui": {
     "port": 80
   },
+  "clusters": {
+    "storage": {
```

Comment:
Just wondering if you have checked if the observed behaviour when these data is passed as "" is the desired one... :)